boards/st/nucleo_h533re/nucleo_h533re.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2024
+# SPDX-License-Identifier: Apache-2.0
+
+identifier: nucleo_h533re
+name: ST Nucleo H533RE
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+ram: 272
+flash: 512
+supported:
+  - gpio
+  - watchdog
+  - pwm
+  - dma
```

Comment:
have you tested `dma`? also for this tag to be included I think an overlay should be added to `tests/drivers/dma/loop_transfer/boards`

Reply:
No i didn't tested the dma, i think should be removed from this PR and treated on sperated one.